website/content/docs/connect/gateways/api-gateway/configuration/routes.mdx

```diff
@@ -212,17 +151,29 @@ The following table describes the parameters for `path`:
 | `replacePrefixMatch` | Specifies a value that replaces the path prefix for incoming HTTP requests. The operation only affects the path prefix. The rest of the path is unchanged. | String | Required |
 | `type` | Specifies the type of replacement to use for the URL path. You can specify the following values: <ul><li>`ReplacePrefixMatch`: Replaces the matched prefix of the URL path (default). </li></ul> | String | Optional |
 
-### rules.matches
+### `rules.filters.extensionRef`
```

Comment:
should we keep a list of links here for out currently supported types? it should be `RouteAuthFilter`, `RouteRetryFilter`, and `RouteTimeoutFilter`

Reply:
Ah, yes. I'll add the missing types to the `kind` row in the table.